Tests

Ninja3047 opened the pull request:
Add a ping test

Ninja3047 committed:
Add a test

asharma-gh reviewed (changes requested):
Let's spin up redis n check that it's running in test helper

Ninja3047 committed:
Implement checks to see if redis server is running

Additional commands will be added later to set up the redis instance
with data for testing
